helpdesk_ticket_multiple_attachments_from_mobile/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Ticket Attachment Enhancer From Mobile",
+    "summary": """
+        Enable multiple fields for ticket creation via mobile to allow uploading multiple photos taken from the device camera
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "17.0.1.0.0",
+    'category': "Helpdesk",
+    "website": "https://github.com/solvosci/slv-helpdesk",
+    "depends": ["helpdesk_mgmt"],
+     "assets": {
+    "web.assets_frontend": [
+        "helpdesk_ticket_multiple_attachments_from_mobile/static/src/js/custom_script.js",
+        ],
+    },
```

Comment:
Make more standard this code formatting:
```suggestion
    "assets": {
        "web.assets_frontend": [
            "helpdesk_ticket_multiple_attachments_from_mobile/static/src/js/custom_script.js",
        ],
    },
```